src/npc/behaviour/ai_behaviour_base.py

```diff
@@ -7,37 +7,35 @@
 from pathfinding.core.grid import Grid
 from pathfinding.finder.a_star import AStarFinder
 
-from src.overlay.soil import SoilLayer
 from src.sprites.entity import Entity
 
 
 # TODO: Refactor NPCState into Entity.state (maybe override Entity.get_state())
-class NPCState(IntEnum):
+class AIState(IntEnum):
```

Comment:
This enum could be used for the player as well, not just CPU-controlled entities

Reply:
You're right, I've been thinking about that too. Yesterday, inspired by raphaels PR, I played around with the entity animation system and changed the raw string references in Entity.frames to enum members. I think that could be used here instead of the AIState as well, although the current way NPC pathfinding works is still dependent on it being separate from the Entity.state. I'm working on changing that, thanks for pointing it out!